engine/dlib/src/dlib/connection_pool.cpp

```diff
@@ -249,80 +251,81 @@ namespace dmConnectionPool
         return false;
     }
 
-    static Result ConnectSocket(HPool pool, dmSocket::Address address, uint16_t port, int timeout, Connection* c, dmSocket::Result* sr)
+    static Result ConnectSocket(HPool pool, dmSocket::Address address, uint16_t port, int timeout, dmSocket::Socket* socket, dmSocket::Result* sr)
```

Comment:
The `Connection* c` is a mutex protected variable, and it wasn't needed for this specific function.